ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -280,12 +281,12 @@ void WaylandWindow::Minimize() {
   xdg_surface_->SetMinimized();
   connection_->ScheduleFlush();
   is_minimized_ = true;
+  was_minimized_ = !is_minimized_;
```

Comment:
nit: I rather be explicit here: was_minimized_ = false;

Reply:
done